test/javascripts/remote_test.coffee

```diff
@@ -94,10 +94,7 @@ describe 'Remote', ->
         fail: done
       , @initiating_target
 
-      simulateError = ->
-        listener(target: remote.xhr) for listener in remote.xhr.eventListeners.error
-
-      setTimeout(simulateError, 0)
+      remote.xhr.respond(404)
```

Comment:
The previous version of this test spit out some distracting error messages (an NPE on xhr).